controllers/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -524,19 +535,25 @@ func (r *Reconciler) sendToolchainStatusNotification(logger logr.Logger,
 }
 
 func ClusterURLs(instance *toolchainv1alpha1.ToolchainStatus) map[string]string {
-	cfg, err := rest.InClusterConfig()
-	if err != nil {
-		result := map[string]string{}
 
-		for _, mbr := range instance.Status.Members {
-			result["Member cluster"] = mbr.ClusterName
+	if instance.Status.HostRoutes.ProxyURL != "" {
+		var domain string
+		var err error
+		if domain, err = removeSchemeFromUrl(instance.Status.HostRoutes.ProxyURL); err != nil {
+			domain = instance.Status.HostRoutes.ProxyURL
```

Comment:
Let's log the error here.

Reply:
I guess it's a small corner case , but I'm wondering if we should still rely on the `.Status.HostRoutes.ProxyURL` when there is an error removing the scheme ( it probably means that the URL is invalid). It shouldn't break the email delivery part. 

Could you please add a unit test for this error case?

Reply:
I guess `domain = instance.Status.HostRoutes.ProxyURL` is a mistake? It's supposed to set domain to the ClusterURL?

Reply:
+1 to adding a unit test for the error case

Reply:
Yeah makes sense.Added a test to pass incorrect url and remove the setting of domain to proxyURL in case of error